sharktank_models/llama3.1/test_llama.py

```diff
@@ -0,0 +1,111 @@
+import iree.compiler
+import iree.runtime
+import numpy
+import os
+import pytest
+import torch
+
+page_size = 12288
+block_size = 16
+
+
+@pytest.fixture
+def llama_mlir():
+    dirname = os.path.dirname(__file__)
+    return f"{dirname}/assets/toy_llama.mlir"
+
+
+@pytest.fixture
+def llama_irpa():
+    dirname = os.path.dirname(__file__)
+    return f"{dirname}/assets/toy_llama.irpa"
```

Comment:
Prefer to use `from pathlib import Path` (https://docs.python.org/3/library/pathlib.html) over `os.path`. This code assumes Unix-style file paths when it doesn't need to.

```suggestion
from pathlib import Path
THIS_DIR = Path(__file__).parent

@pytest.fixture
def llama_mlir():
    return str(THIS_DIR  / "assets" / "toy_llama.mlir")


@pytest.fixture
def llama_irpa():
    return str(THIS_DIR  / "assets" / "toy_llama.irpa")
```

(You can either return a `Path` or a string. I'd prefer to return a `Path` and only convert to string where necessary)

Reply:
Fixed.